package.json

```diff
@@ -21,6 +21,7 @@
     "calculate-external-asset-info": "ts-node 'scripts/calculate-external-asset-info.ts'",
     "calculate-local-asset-info": "ts-node 'scripts/calculate-local-asset-info.ts'",
     "calculate-multilocation-derivative-account": "ts-node 'scripts/calculate-multilocation-derivative-account.ts'",
+    "calculate-remote-origin": "ts-node 'scripts/calculate-multilocation-derivative-account.ts'",
```

Comment:
Same script as the line above?

Reply:
Yeah, two ways of calling it just because the first one is `legacy` and the second one is using the actual terminology 😄